packages/expo-location/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Actually, I don't think this will work seamlessly - we would need to change `rootDir` of the `expo/tsconfig.json` file to be `..` instead of `./src` and then universal modules will be compiled into `packages/expo/build` folder and I guess it's not what we're going to achieve. Can I keep these configs for now until we find a good solution?